src/test/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/UserApiTest.java

```diff
@@ -448,7 +449,7 @@ public void testPasswordRules() throws Exception {
         Assert.assertTrue(response.getBody().contains("xxx"));
 
         response = rh.executePutRequest("/_opendistro/_security/api/internalusers/ok1", "{\"backend_roles\":[\"my-backend-role\"],\"attributes\":{},\"password\":\"\"}", new Header[0]);
-        Assert.assertEquals(HttpStatus.SC_OK, response.getStatusCode());
+        Assert.assertEquals(HttpStatus.SC_BAD_REQUEST, response.getStatusCode());
```

Comment:
Do include test for empty password but fix request instead of response for the tests as they are asserting updating other attributes

Reply:
Fixed the response for expected behavior change. And keep the original test case below.